### PR TITLE
Salutation and Gender in Lead and Customer

### LINKED ISSUE #8145: [Enhancement] Salutation in lead & contact name
Suggestion:
- Add salutation field to lead
- Add salutation field to contact
- Map salutation lead field to contact when lead is converted to customer

Benefit:
- Field can be used in sales quotations (skipping salutation might seem a little too direct in some cultures, esp. when first name is not available for a contact)
- Could potentially be used for analytics (filter by gender for instance)
- Field can be used in purchase orders, RFQ

### PR DESCRIPTION
**Pull request by KanchanChauhan**
Fixed https://github.com/frappe/erpnext/issues/8145 and https://github.com/frappe/erpnext/issues/8143

screen shot 2017-04-12 at 2 43 12 pm

### CODE REVIEW

File: erpnext/crm/doctype/lead/lead.json [hunk 1]
```diff
@@ -106,21 +136,21 @@
...
-   "in_list_view": 1, 
+   "in_list_view": 0, 
    "in_standard_filter": 0, 
-   "label": "Organization Name", 
+   "label": "Gender", 
    "length": 0, 
    "no_copy": 0, 
-   "oldfieldname": "company_name", 
-   "oldfieldtype": "Data", 
+   "options": "\nMale\nFemale",
```
Comment (rmehta) [inline erpnext/crm/doctype/lead/lead.json]: other?

**Review by rmehta — CHANGES_REQUESTED** [review]

File: erpnext/crm/doctype/lead/lead.json (hunk 1 above)
Comment (KanchanChauhan) [inline erpnext/crm/doctype/lead/lead.json]: If we do others, we need to give text box to mention the other gender. Blank field is better I think.

Comment (rmehta) [inline erpnext/crm/doctype/lead/lead.json]: blank is unknown / not set ... other is other!

Comment (KanchanChauhan) [inline erpnext/crm/doctype/lead/lead.json]: Done

**Comment by nabinhait** [comment]
Closing this as it depends on https://github.com/frappe/frappe/pull/3079 which is closed.

**Pull request closed without merging**